includes/modules/bootstrap/centerboxes/also_purchased_products.php

```diff
@@ -49,7 +49,7 @@
             if (empty($also_purchased_products->fields['products_image']) && (int)PRODUCTS_IMAGE_NO_IMAGE_STATUS === 0) {
                 $also_purchased_products_image = '';
             } else {
-                $also_purchased_products_image = '<div class="centerBoxContentsItem-image text-center"><a href="' . $app_products_link . '">' . zen_image(DIR_WS_IMAGES . $also_purchased_products->fields['products_image'], $app_products_name, SMALL_IMAGE_WIDTH, SMALL_IMAGE_HEIGHT) . '</a></div>';
+                $also_purchased_products_image = '<div class="centerBoxContentsItem-image text-center"><a href="' . $app_products_link . '" title="' . $app_products_name . '">' . zen_image(DIR_WS_IMAGES . $also_purchased_products->fields['products_image'], $app_products_name, SMALL_IMAGE_WIDTH, SMALL_IMAGE_HEIGHT) . '</a></div>';
```

Comment:
Need to change `$app_products_name` to `zen_output_string_protected($app_products_name)`; otherwise, products with HTML special characters in their names will result in invalid HTML being rendered.

Reply:
In hornedlizardcreations.com --

Select Texas Horned Lizard from the Categories Sidebox, Select the image for Texas Horned Lizard, then Hand Painted Figurines.

Scroll down to Also Purchased.  You may have to refresh the page several times but, make one of the choices "Texas Horned Lizard's Pewter & Key Chains.

Inspecting shows it had no problem with the apostrophe or the ampersand. 

HTML Validator is okay with it as well.

The HTML validator is complaining about no alt tags in ceterbox items.

I think I need to look at this further to make sure both HTML and ADA/WCAG are satisfied so we don't keep going in circles.